HMMVoiceCreation.md

```diff
@@ -76,9 +168,9 @@ And run the following components:
 
 2- Run the **EHMMlabeler** to label automatically the wav files using the corresponding transcriptions. If the pauses at the beginning and end of your recordings are longer than 0.2 seconds, you might consider to reduce these pauses using the tool: Convert recorded audio (as explained in NewLanguageSupport No. 9) to trim initial and final silences.
```

Comment:
I think it would be better to drop Section III entirely -- it should be covered by the main VoiceImportTools page (maybe add a note at the top)